README.md

```diff
@@ -34,14 +46,65 @@ To run with the nginx server (on a Mac):
 To stop the server:
   - Option 1: `sudo nginx -c /usr/local/etc/nginx/nginx.conf -s stop`
   - Option 2: `sudo nginx -s stop`
+  
+## Using docker-compose
 
-## Docker
+### Starting the Front-end
 
-To run with docker:
-1. Build the production version of this project: `ng build --prod`
-2. Build image: `docker build -t angular-nginx .`
-3. Run container: `docker run --name angular-nginx-container -d -p 8090:80 angular-nginx`
-4. Go to `localhost:8090`
+**If you have no containers** in your environment from a previous run of docker-compose, run: 
+
+```bash
+docker-compose up
+```
+
+To start the service in the background, add `-d` to your `docker-compose up` command.
+
+**If you already have containers** in your environment from a previous run of docker-compose, run: 
+
+```bash
+docker-compose start
+```
+
+### Stopping the Front-end
+
+To stop the service and retain the container, run:
+
+```bash
+docker-compose stop
+```
+
+To stop the service and remove the container, run:
+
+```bash
+docker-compose down
+```
+
+To stop the service, remove the container and image, run:
+
+```bash
+docker-compose down --rmi all -v
+```
+
+### Starting Services Behind SSL-Inspecting Proxy
+If you are operating behind a proxy that does SSL-Inspection, yarn needs to be configured with the certificates that are used by the proxy in the image. Currently, `docker-compose.yml` includes the location of a `certs` directory as an argument to the `Dockerfile`. If you are operating behind an SSL-Inspecting proxy **you will have to copy your certificates into this directory.**
```

Comment:
You should mention that it is expected that the root certificate will exist in a file named `ca-certificates.crt` since there is the line in the Dockerfile that expects that file to exist

Reply:
👍